build.gradle

```diff
@@ -15,14 +15,14 @@ buildscript {
         classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.0'
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.3.1'
         classpath 'org.jfrog.buildinfo:build-info-extractor-gradle:4.4.12'
-        classpath 'com.palantir.gradle.javadist:gradle-java-distribution:1.3.0'
         classpath 'gradle.plugin.com.palantir.gradle.docker:gradle-docker:0.8.0'
         classpath 'com.palantir.baseline:gradle-baseline-java:0.13.0'
         classpath 'com.palantir:jacoco-coverage:0.4.0'
         classpath "com.netflix.nebula:gradle-dependency-lock-plugin:4.3.2"
         classpath 'com.netflix.nebula:nebula-dependency-recommender:3.6.3'
         classpath 'com.netflix.nebula:nebula-publishing-plugin:4.9.1'
         classpath 'nl.javadude.gradle.plugins:license-gradle-plugin:0.11.0'
+        classpath "com.palantir.sls-packaging:gradle-sls-packaging:2.0.0"
```

Comment:
curious why not use latest 2.3.0 per https://github.com/palantir/sls-packaging/releases?